tor/strings/en_US.yml

```diff
@@ -79,14 +79,6 @@ responses:
       ✅ The post is yours! Best of luck and thanks for helping!
 
       Does this post break the rules? Please report it! If you don't want this post anymore (or can't complete it), respond to this comment with `unclaim`.
-
```

Comment:
We need to leave an empty line between those so that we get the line break on Reddit